n3fit/src/evolven3fit_new/utils.py

```diff
@@ -134,6 +134,8 @@ def generate_q2grid(Q0, Qfin, Q_points, match_dict):
         Q_ini = Q0
         num_points_list = []
         for masses in match_dict:
+            # match_dict contains the couples (mass : factor) where factor is the number 
+            # to be multiplied to mass in order to obtain the relative matching scale.
```

Comment:
Maybe lift this explanation in the docstring. It is coming from outside, and will be accessible in more places (including built-in `help()`).